### ISSUE TITLE
Enable https over Tailscale

### ISSUE THREAD

**Issue by exorcist-transfix-welsh** [getumbrel/umbrel#1514]
Many Umbrel apps are currently not fully functional, as certain functions require access over https. Vaultwarden, Specter Desktop, LNbits and others.

If the Tailscale app could be configured to allow https connections, then many apps will benefit.

**Comment by eurbasius** [comment]
Hey. Have you looked into the Tailscale? It could be a partial solution.

**Comment by eurbasius** [comment]
Hey. Have you looked into it? https://tailscale.com/kb/1153/enabling-https/

**Comment by exorcist-transfix-welsh** [comment]
I’ve tried following this guide, but something about the way umbrel is setup to serve things isn’t playing nicely with how tailscale expects things to be.

**Comment by eurbasius** [comment]
I agree with you. I have a lot of problems. Chose to wait instead. Maybe they will address it later.

**Comment by mjboroski** [comment]
The problem is that the `tailscale cert` command needs to be run inside the docker container running tailscale on umbrel. That's all well and good, but then we need to wonder if the way umbrel does app updates (by removing the container and installing the newer one) will make it so every update requires this to be redone. We also need to question if we should be storing this cert in the docker container at all. I'm not smart enough on this topic to solve the issue, but I can point the relevant people to a link I found that I think solves this... I think. https://github.com/tailscale/tailscale/issues/5072#issuecomment-1214334796

**Comment by satoshinotdead** [comment]
There is some update here?

I'm facing the same issue but I suspect I will end doing things manually and stop using Umbrel plus Tailscale.

**Comment by B0G0311** [comment]
Any update on this request?

**Comment by only1thor** [comment]
Looks like the `tailscale serve` command can be a good fit for a quick toggle to enable https over tailscale:
https://tailscale.com/kb/1242/tailscale-serve
